### PR TITLE
Add org.gtk.Gtk3theme.Akwa-dark

### PR DESCRIPTION
**Pull request by berkiyo**

### CODE REVIEW

**Comment by berkiyo** [comment]
bot, build org.gtk.Gtk3theme.Akwa

**Comment by berkiyo** [comment]
oops. :D

**Comment by berkiyo** [comment]
bot, build org.gtk.Gtk3theme.Akwa-dark

File: org.gtk.Gtk3theme.Akwa-dark.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "id": "org.gtk.Gtk3theme.Akwa-dark",
+    "branch": "3.22",
+    "runtime": "org.freedesktop.Platform",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
```
Comment (barthalion) [inline org.gtk.Gtk3theme.Akwa-dark.json:7]:
```suggestion
    "runtime-version": "20.08",
```

**Comment by berkiyo** [comment]
Thanks @barthalion, still new to Flatpak-ing!

**Comment by berkiyo** [comment]
bot, build org.gtk.Gtk3theme.Akwa-dark

**Comment by barthalion** [comment]
bot, build org.gtk.Gtk3theme.Akwa-dark

**Comment by barthalion** [comment]
Fat fingers, sorry.

**Comment by barthalion** [comment]
/merge

All looks good, sorry for terse review comments but I've been catching up with new submissions. Thanks!

**Pull request closed without merging**